pkg/cli/jobflow/get.go

```diff
@@ -0,0 +1,96 @@
+package jobflow
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"volcano.sh/apis/pkg/apis/flow/v1alpha1"
+	"volcano.sh/apis/pkg/client/clientset/versioned"
+	"volcano.sh/volcano/pkg/cli/util"
+)
+
+type getFlags struct {
+	util.CommonFlags
+	// Name of the jobflow
+	Name string
+	// Namespace of the jobflow
+	Namespace string
+}
+
+var getJobFlowFlags = &getFlags{}
+
+// InitGetFlags is used to init all flags.
+func InitGetFlags(cmd *cobra.Command) {
+	util.InitFlags(cmd, &getJobFlowFlags.CommonFlags)
+	cmd.Flags().StringVarP(&getJobFlowFlags.Name, "name", "N", "", "the name of jobflow")
+	cmd.Flags().StringVarP(&getJobFlowFlags.Namespace, "namespace", "n", "default", "the namespace of jobflow")
+}
+
+// GetJobFlow gets a jobflow.
+func GetJobFlow(ctx context.Context) error {
+	config, err := util.BuildConfig(getJobFlowFlags.Master, getJobFlowFlags.Kubeconfig)
+	if err != nil {
+		return err
+	}
+
+	if getJobFlowFlags.Name == "" {
+		err := fmt.Errorf("name is mandatory to get the particular jobflow details")
+		return err
+	}
+
+	jobFlowClient := versioned.NewForConfigOrDie(config)
+	jobFlow, err := jobFlowClient.FlowV1alpha1().JobFlows(getJobFlowFlags.Namespace).Get(ctx, getJobFlowFlags.Name, metav1.GetOptions{})
```

Comment:
If `getJobFlowFlags.Namespace` == "", it will return all jobflow, we should set a default ns here.